radixdlt/src/main/java/com/radixdlt/consensus/VoteData.java

```diff
@@ -0,0 +1,78 @@
+package com.radixdlt.consensus;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.radixdlt.serialization.DsonOutput;
+import com.radixdlt.serialization.DsonOutput.Output;
+import com.radixdlt.serialization.SerializerConstants;
+import com.radixdlt.serialization.SerializerDummy;
+import com.radixdlt.serialization.SerializerId2;
+import java.util.Objects;
+import java.util.Optional;
+import javax.annotation.concurrent.Immutable;
+
+@Immutable
+@SerializerId2("consensus.vote_data")
+public final class VoteData {
```

Comment:
Document me, please. Would also be useful to have an overview of the relationship between VoteData, VertexMetadata, Vertex, etc. for outsiders and future maintainers.